assets/src/blocks/Media/MediaEditor.js

```diff
@@ -0,0 +1,155 @@
+import { Fragment, useEffect, useState, useCallback } from "@wordpress/element";
+import { Tooltip, PanelBody } from '@wordpress/components';
+import { MediaPlaceholder } from "@wordpress/editor";
+import { InspectorControls } from '@wordpress/block-editor';
+
+import { URLInput } from "../../components/URLInput/URLInput";
+import { MediaEmbedPreview } from "./MediaEmbedPreview";
+import { MediaElementVideo } from './MediaElementVideo';
+import { useSelect } from '@wordpress/data';
+import { debounce } from 'lodash';
+
+const { __ } = wp.i18n;
+const { RichText } = wp.blockEditor;
+
+const renderEdit = ({ attributes, setAttributes }) => {
+  // Using a state to prevent the input losing the cursor position, a React issue reported multiple times
+  const [ mediaURL, setMediaURL ] = useState(attributes.youtube_id);
+  const debouncedMediaURLUpdate = useCallback(debounce(value => setAttributes({ youtube_id: value }), 300), []);
+
+  return (
+    <Fragment>
```

Comment:
I don't think the `Fragment` is needed here, since there is only one child...? 🤔